File: 8kyu/count-of-positives-sum-of-negatives/count-of-positives-sum-of-negatives.py
def count_positives_sum_negatives(arr):
  if not arr:
    return []

  positive, negative = 0, 0
  for num in arr:
    if num > 0:
      positive+=1
    else:
      negative += num
  return [positive, negative]

print(count_positives_sum_negatives([1, 2, 3, 4, 5, 6, 7, 8, 9, 10, -11, -12, -13, -14, -15]))
print(count_positives_sum_negatives([]))

File: 8kyu/count-of-positives-sum-of-negatives/count-of-positives-sum-of-negatives.ts
export function countPositivesSumNegatives(input: any) {
	if (!input?.length) return [];

	let [positive, negative] = [0, 0];
	for (const num of input) {
		if (num > 0) positive++;
		if (num < 0) negative += num;
	}

	return [positive, negative];
}
